tests/foreman/maintain/test_advanced.py

```diff
@@ -369,7 +378,7 @@ def test_positive_capsule_repositories_setup(sat_maintain):
 
     :expectedresults: Required Capsule repositories should get enabled
     """
-    supported_versions = ['6.11', '6.12']
+    supported_versions = ['6.11', '6.12', '6.13']
```

Comment:
I'm really not a huge fan of this test. We should only run the version on its corresponding branch, and I really think there's no reason to test non-GA repo set up. During my audit of FM, I considered removing this test or at least modifying it to the above. What do you think?